python-base/ubuntu20.04-python3.9-nginx/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM digitallivinginternational/python-base:ubuntu20.04-python3.9
```

Comment:
I'd prefer it if this didn't require the username part, it might be necessary to update the build script so it also includes the tag based on the directory alone without the username bit - no need to do that for the aliases tho